SConscript

```diff
@@ -217,8 +217,8 @@ if env['PLATFORM'] == 'darwin':
 # link site-packages/python*/*.so -> *.so to allow python import
 if env['PLATFORM'] == 'posix':
     for  f in libmstruct:
-    	 if f.get_suffix()=='.so':
-	    link_target = os.path.normpath( os.path.join(env['modulepath'], f.rstr()) )
+        if f.get_suffix()=='.so':
```

Comment:
spaces around `==`

Reply:
Hi thanks, but we still need line 220. Seems you omitted it.